.circleci/config.yml

```diff
@@ -460,6 +460,62 @@ jobs:
           branch_base: en
           commit_message: "Update deployment info for << parameters.chain-id >>-<< parameters.preset >>"
 
+  deployments-dashboard:
+    docker:
+      - image: cimg/node:<< pipeline.parameters.node-version >>
+    steps:
+      - checkout
+      - yarn-install
+
+      - run: mkdir -p ./e2e/cannon
+
+      - run: yarn cannon inspect synthetix-omnibus:latest@andromeda   --chain-id 8453   --json > ./e2e/cannon/8453-andromeda.json
+      - run: yarn cannon inspect synthetix-omnibus:latest@andromeda   --chain-id 84532  --json > ./e2e/cannon/84532-andromeda.json
+      - run: yarn cannon inspect synthetix-omnibus:latest@main        --chain-id 1      --json > ./e2e/cannon/1-main.json
+      - run: yarn cannon inspect synthetix-omnibus:latest@main        --chain-id 10     --json > ./e2e/cannon/10-main.json
+      - run: yarn cannon inspect synthetix-omnibus:latest@arbthetix   --chain-id 42161  --json > ./e2e/cannon/42161-arbthetix.json
+
+      - store_artifacts:
+          path: "./e2e/cannon"
+
+      - store_artifacts:
+          path: "./e2e/deployments"
+
+      - run:
+          name: "Generate docs"
+          environment:
+            DEBUG: "e2e:*"
+          command: |
+            mkdir -p docs/for-developers
+            node e2e/dashboard > docs/for-developers/deployment-info.md
```

Comment:
Same thing here, shouldn't this be `.html`

Reply:
it is md not html.